hera_cal/abscal_funcs.py

```diff
@@ -1676,6 +1667,7 @@ def avg_file_across_red_bls(data_fname, outdir=None, output_fname=None,
 
     # write to file
     if write_miriad:
+        raise NotImplementedError("Correct averaging of integration_time and nsample not yet implemented")
```

Comment:
This is a bit strange. You definitely want to comment out the code that isn't usable. But I wonder: why aren't we using `io.write_vis`? Can't we just eliminate this whole block?

Reply:
I punted on writing b/c we need to implement the proper summing of the `nsample` and `integration_time` arrays which is not trivial and I didn't want to invest time into it now. I can comment out that code if you like

Reply:
Yeah, I'd comment out the code. Otherwise our coverage will go way down.

Alternatively, if no one is using the code, we could also delete it.